src/compute/src/server.rs

```diff
@@ -283,7 +284,7 @@ pub async fn compute_node_serve(
     // - https://github.com/risingwavelabs/risingwave/issues/8696
     // - https://github.com/risingwavelabs/risingwave/issues/8822
     let memory_mgr = GlobalMemoryManager::new(
-        opts.total_memory_bytes,
+        compute_memory_bytes,
```

Comment:
It seems that the 1st parameter should be `compute_memory_bytes` instead of `total`
```rust
    pub fn new(
        total_compute_memory_bytes: usize,
        barrier_interval_ms: u32,
        metrics: Arc<StreamingMetrics>,
        memory_control_policy: MemoryControlPolicy,
    )
```